test/web-service/web-service.test.ts

```diff
@@ -737,16 +737,9 @@ describe("WebService", () => {
       ],
 
       // Would not allow evicting any replicas
-      [{ replicas: 2 }, { minAvailable: IntOrString.fromNumber(2) }],
       [{ replicas: 2 }, { maxUnavailable: IntOrString.fromNumber(0) }],
-      [{ replicas: 3 }, { minAvailable: IntOrString.fromString("100%") }],
       [{ replicas: 3 }, { maxUnavailable: IntOrString.fromString("0%") }],
-      [{ replicas: 10 }, { minAvailable: IntOrString.fromString("91%") }],
       [{ replicas: 42 }, { maxUnavailable: IntOrString.fromString("0%") }],
-      [
-        { minReplicas: 5, maxReplicas: 10 },
-        { minAvailable: IntOrString.fromString("85%") },
-      ],
```

Comment:
Are these removed for a reason?

Reply:
Yes, these were expected to fail before. Now, after validation condition is changed, these 4 test cases are failing because `replicas == minAvailable`. That's why they are removed.

Reply:
These tests are checking cases when validation does fail. So, for example, case

```
[{ replicas: 2 }, { minAvailable: IntOrString.fromNumber(2) }],
```

should still fail, because if we only have 2 pods, and PDB enforces that there must be 2, we would get eviction errors.

So I think the change to `if (min > minReplicas)` check is wrong, it should be `>=` (error if PDB's minAvailable is equal to minReplicas).

Reply:
That reverts it to the initial condition.